lib/plugins/aws/provider.js

```diff
@@ -26,6 +32,8 @@ const constants = {
   providerName: 'aws',
 };
 
+const imageNamePattern = '^[a-z][a-z0-9-_]{1,31}$';
```

Comment:
Is upper case not allowed for a reason? Are those chars not supported on AWS side (as it's with e.g. bucket names) ?

Reply:
It's not allowed on AWS side to use uppercase letters in repository names. I've found that the hard way at the beginning where I allowed it and had my repository creation crash due to that very reason :+1:

Reply:
Ok 👍